Plaid API Endpoints Collection.json

```diff
@@ -5678,6 +5678,605 @@
+								}
+							]
+						}
+					]
+				},
+				{
+					"name": "Get a transfer",
+					"request": {
+						"method": "POST",
+						"header": [
+							{
+								"key": "Content-Type",
+								"value": "application/json"
+							}
+						],
+						"body": {
+							"mode": "raw",
+							"raw": "{\n    \"client_id\": \"{{client_id}}\",\n    \"secret\": \"{{secret_key}}\",\n    \"transfer_id\": \"ENTER_TRANSFER_ID_HERE\"\n}"
+						},
+						"url": {
+							"raw": "https://{{env_url}}/transfer/get",
+							"protocol": "https",
+							"host": [
+								"{{env_url}}"
+							],
+							"path": [
+								"transfer",
+								"get"
+							]
+						},
+						"description": "The `/transfer/get` fetches information about the transfer corresponding to the given `transfer_id`."
+					},
+					"response": [
+						{
+							"name": "OK",
+							"originalRequest": {
+								"method": "POST",
+								"header": [],
+								"body": {
+									"mode": "raw",
+									"raw": "{\n    \"client_id\": \"<string>\",\n    \"secret\": \"<string>\",\n    \"transfer_id\": \"<string>\"\n}"
+								},
+								"url": {
+									"raw": "{{baseUrl}}/transfer/get",
+									"host": [
+										"{{baseUrl}}"
+									],
+									"path": [
+										"transfer",
+										"get"
+									]
+								}
+							},
+							"status": "OK",
+							"code": 200,
+							"_postman_previewlanguage": "json",
+							"header": [
+								{
+									"key": "Content-Type",
+									"value": "application/json"
+								}
+							],
+							"cookie": [],
+							"body": "{\n  \"transfer\": {\n    \"account_id\": \"6qL6lWoQkAfNE3mB8Kk5tAnvpX81qefrvvl7B\",\n    \"ach_class\": \"ppd\",\n    \"amount\": \"12.34\",\n    \"cancellable\": true,\n    \"created\": \"2020-08-06T17:27:15Z\",\n    \"description\": \"Testing2\",\n    \"failure_reason\": {\n      \"ach_return_code\": \"R13\",\n      \"description\": \"Invalid ACH routing number\"\n    },\n    \"id\": \"460cbe92-2dcc-8eae-5ad6-b37d0ec90fd9\",\n    \"metadata\": {\n      \"key1\": \"value1\",\n      \"key2\": \"value2\"\n    },\n    \"network\": \"ach\",\n    \"origination_account_id\": \"11111111-1111-1111-1111-111111111111\",\n    \"status\": \"pending\",\n    \"type\": \"credit\",\n    \"user\": {\n      \"email_address\": \"plaid@plaid.com\",\n      \"legal_name\": \"John Smith\",\n      \"phone_number\": null,\n      \"address\": null\n    }\n  },\n  \"request_id\": \"saKrIBuEB9qJZno\"\n}"
+						}
+					]
+				},
+				{
+					"name": "Create a transfer",
```

Comment:
```suggestion
					"name": "Initiate a transfer",
```

Reply:
It's what we say in the docs, and there's `create` all over the transfer endpoints, so I thought this might be a little less confusing. Minor though, don't have to incorporate it.